third_party/move/move-model/Cargo.toml

```diff
@@ -8,23 +8,20 @@ license = "Apache-2.0"
 
 [dependencies]
 anyhow = { workspace = true }
-move-binary-format = { path = "../move-binary-format" }
-move-bytecode-source-map = { path = "../move-ir-compiler/move-bytecode-source-map" }
-move-command-line-common = { path = "../move-command-line-common" }
-# diem dependencies
-move-compiler = { path = "../move-compiler" }
-move-core-types = { path = "../move-core/types" }
-move-disassembler = { path = "../tools/move-disassembler" }
-move-ir-types = { path = "../move-ir/types" }
-move-symbol-pool = { path = "../move-symbol-pool" }
-
-# external dependencies
 codespan = { workspace = true }
 codespan-reporting = { workspace = true }
 either = { workspace = true }
 internment = { workspace = true, features = ["arc"] }
 itertools = { workspace = true }
 log = { workspace = true }
+move-binary-format = { workspace = true }
+move-bytecode-source-map = { workspace = true }
+move-command-line-common = { workspace = true }
+move-compiler = { workspace = true }
+move-core-types = { workspace = true }
+move-disassembler = { workspace = true }
+move-ir-types = { workspace = true }
+move-symbol-pool = { workspace = true }
 num = { workspace = true }
 num-traits = { workspace = true }
 once_cell = { workspace = true }
```

Comment:
Could be made `{ workspace = true }` for `move-prover-test-utils` below?

Reply:
Replied above